common/types/src/monitoring.rs

```diff
@@ -0,0 +1,122 @@
+use std::{collections::HashSet, sync::LazyLock, time::SystemTime};
+
+use nym_crypto::asymmetric::identity::{PrivateKey, PublicKey, Signature};
+use nym_mixnet_contract_common::MixId;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+static NETWORK_MONITORS: LazyLock<HashSet<String>> = LazyLock::new(|| {
+    let mut nm = HashSet::new();
+    nm.insert("5VsPyLbsBCq9PAMWmjKkToteVAKNabNqex6QwDf5fWzt".to_string());
+    nm
+});
+
+#[derive(Debug, Serialize, Deserialize, JsonSchema, Clone)]
+pub struct MixnodeResult {
+    pub mix_id: MixId,
+    pub identity: String,
+    pub owner: String,
+    pub reliability: u8,
+}
+
+impl MixnodeResult {
+    pub fn new(mix_id: MixId, identity: String, owner: String, reliability: u8) -> Self {
+        MixnodeResult {
+            mix_id,
+            identity,
+            owner,
+            reliability,
+        }
+    }
+}
+
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+pub struct GatewayResult {
+    pub identity: String,
```

Comment:
it will save me a lot of headache in rebasing if you also included `mix_id` / `node_id` here

maybe just have a 

```rust
pub struct NodeResult {
    pub node_id: MixId,
    pub identity: String,
    pub owner: String,
    pub reliability: u8,
}
```

and it would just temporarily be `0` for gateways : )

because the idea is to unify gateways and mixnodes and remove as many "node-specific" features as possible